infrastructure/lambda-bulk-upload-report.tf

```diff
@@ -0,0 +1,23 @@
+module "bulk-upload-report-lambda" {
+  source  = "./modules/lambda"
+  name    = "BulkUploadReportLambda"
+  handler = "handlers.bulk_upload_report_handler.lambda_handler"
+  iam_role_policies = [
+    "arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole",
+    "arn:aws:iam::aws:policy/CloudWatchLambdaInsightsExecutionRolePolicy",
+    module.ndr-bulk-staging-store.s3_object_access_policy,
+  ]
+  rest_api_id       = aws_api_gateway_rest_api.ndr_doc_store_api.id
+  api_execution_arn = aws_api_gateway_rest_api.ndr_doc_store_api.execution_arn
+  lambda_environment_variables = {
+    WORKSPACE                 = terraform.workspace
+    STAGING_STORE_BUCKET_NAME = "${terraform.workspace}-${var.staging_store_bucket_name}"
```

Comment:
Same as above, if need to read from bulk_upload_dynamodb_table, than we could pass the table name from env var